src/network/receive.ts

```diff
@@ -244,7 +262,7 @@ async function onReceive(payload: Payload, dest: string) {
           owner,
           forwardedFromContactId
         )
-      else sphinxLogger.error(`=> insufficient payment for this action`)
+      } else sphinxLogger.error(`=> insufficient payment for this action`)
```

Comment:
lets try adding network to the end of this since its in a network folder
```sphinxLogger.error(`=> insufficient payment for this action`, logging.network)```